.github/workflows/build-test.yml

```
@@ -55,6 +55,8 @@ jobs:
     - name: Install dependencies
       run: |
         pip install .[docs,tests]
+        pip uninstall --yes huggingface_hub
+        pip install huggingface_hub=="0.11.0rc0"
```

```suggestion
        pip install huggingface_hub=="0.11.0rc0"
        pip install .[docs,tests]
```

more like this one, it'd also make it faster. (I don't think it matters if we're not keeping this in the CI though, was thinking more in the direction of adding it permanently to the CI).

---

Yeah, I just quickly added it, if we choose to create a task for it, your suggestion would be cleaner.